sites/friday/src/get-started/here/make-changes.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Make changes to your project
+sidebarTitle: Revisions
+weight: 50
+description: How to work on a daily basis with an {{% vendor/name %}} project?
+---
+
+Upsun allows you to make changes to your project and test them on a preview environment before introducing them to Production.
+
+In your project, the default branch (e.g. `main`, `master`, whichever chosen during project creation) always represents the production environment.
+Other branches are for developing new features, fixing bugs, or updating the infrastructure.
+
+To make changes to your project, follow these steps:
+
+## 1. Create a new environment
+Create a new environment (a Git branch) to make changes without impacting production:
+{{< codetabs >}}
++++
+title={{% vendor/name %}} Git repository
++++
+
+Run the following command:
+   ```bash {location="Terminal"}
+   {{% vendor/cli %}} branch feat-a
+   ```
+   This creates a new local `feat-a` Git branch based on the `main` Git branch
+   and activates a related environment on {{< vendor/name >}}.
+   The new environment inherits the data (service data and assets) of its parent environment (the production environment here).
+<--->
++++
+title=Using third party provider
++++
+   {{% vendor/name %}} provides a feature called [source integration](integrations/source.html) that allows your {{% vendor/name %}} project to be fully integrated with your external repository.
+   This enables you, as a developer, to use a normal Git workflow (`git add . && git commit -m "message" && git push`) to deploy your environment—with no need to connect to the {{% vendor/name %}} Console.
+
+   Assuming you've already activated the source integration feature on your project, create a new branch:
+   ```bash {location="Terminal"}
+   git checkout -b feat-a && git push -u origin feat-a
+   ```
+   This creates a new local `feat-a` Git branch based on the `main` Git branch
+   and a related environment on {{< vendor/name >}}.
+   The new environment is inactive by default. Once you activate it, it will inherit the data (service data and assets) of its parent environment (the Production environment here).
+
+   {{< note >}}
+   When your local feature is ready, [activate your environment](/administration/cli/reference.html#environmentactivate).
+   {{< /note >}}
+
+{{< /codetabs >}}
+
+{{< note title="Warnings" theme="warning">}}
+   **First**, you are only charged for your [activated environments](/administration/cli/reference.html#environmentactivate).
```

Comment:
Re-reading this again, we should probably link to the definition of an "active environment" BUT the definition we have currently seems incomplete. 
> An environment that’s deployed. 

But what does that mean? 
```suggestion
   **First**, you are charged for all [activated environments](/administration/cli/reference.html#environmentactivate).
```